packages/react-instantsearch-core/src/widgets/InstantSearch.tsx

```diff
@@ -5,7 +5,26 @@ import createInstantSearchManager from '../core/createInstantSearchManager';
 import { InstantSearchProvider, InstantSearchContext } from '../core/context';
 import { Store } from '../core/createStore';
 import { PlainSearchParameters, SearchParameters } from 'algoliasearch-helper';
-import { SearchResponse } from '@algolia/client-search';
+
+import algoliasearch, {
```

Comment:
I feel like this block fits better in its own file in /types (if we have one), like in InstantSearch.js